entity-server-api/src/main/java/org/terracotta/entity/PassiveServerEntity.java

```diff
@@ -36,9 +36,15 @@
    *  under a given key, the messages it creates and sends for synchronization will be executed in that key, on the passive,
    *  via this invoke() entry-point.</p>
    *
+   * @param clientDescriptor source instance from which the invocation originates.
+   * @param currentOrderedId current id this invoke is part of
+   * @param eldestOrderedId earliest active id for this client
    * @param message The message from a client or upstream active server
    */
-  void invoke(M message) throws EntityUserException;
+  void invoke(ClientDescriptor clientDescriptor,
```

Comment:
Is it possible to have an example use case to see how to properly use the new 3 parameters ? I don't really understand how I would use this ClientDescriptor from a passive entity and I am wondering how to adapt the voltron-proxy the best way to support that.

1. what I can do, what I cannot do ? 
2. Is it the same (i.e. equals / hashcode) as the one on the active ? 
3. Do the passive entity will also be aware of the join and leave of this client descriptor ?
4. Do I have the ability to use it to store some things related to this descriptor, and have a way to clean that (meaning knowing that the descriptor is gone) ?
5. On a failover, for example, does the ClientDescriptor will remain the same on the new active as it was observed on the passive that became active ?